navigation/client/navigationLinks.js

```diff
@@ -29,3 +25,10 @@ Template._navigationLinks.helpers({
     showOrHideExportCSVButton: function() {
       return isPage('responses') ? '' : 'hide'; }
 });
+
+Template._navigationLinks.events({
+    "click .share-form": function () {
+        // Tell Sandstorm to show the Share Access menu.
+        window.parent.postMessage({startSharing: {}}, "*");
```

Comment:
I chose `{}` because it's more forward-compatible.  I can add an `rpcId` field to `{}` if we later decide we need to reply to this with e.g. whether we successfully opened the share menu.  I can add additional fields, like a suggested role assignment to `{}` that allows the app to reduce the number of actions the user has to take.

Also, for consistency with the rest of our `postMessage` APIs, I think `{}` is the appropriate "no non-default options" value.